wrangler-core/src/main/java/io/cdap/wrangler/utils/ReferenceNameUtils.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.wrangler.utils;
+
+import java.util.regex.Pattern;
+
+/**
+ * Utility methods for reference name
+ */
+public class ReferenceNameUtils {
```

Comment:
nit: our utility classes don't normally have 'Util' in the name. This would be ReferenceNames